solutions/devpsrint-michael-douglas-1/GitHubApp/Service/Service.swift

```diff
@@ -7,10 +7,42 @@
 
 import Foundation
 
-struct Service {
-
-    func fetchList(_ completion: ([String]) -> Void) {
+protocol ServiceProtocol {
+    
+    func fetchRepositoryList(_ userName: String, _ completion: @escaping ([Repository]?) -> Void)
```

Comment:
Utilizando boas práticas de nomenclatura de métodos, você pode utilizar dessa forma aqui
```suggestion
    func fetchRepositories(fromUserName userName: String, completion: @escaping ([Repository]?) -> Void)
```